tests/letstest/multitester.py

```diff
@@ -594,10 +603,7 @@ class FabricException(Exception):
             sys.exit(1)
 
     finally:
-        cleanup(cl_args, instances, targetlist)
-
-        # kill any connections
-        fabric.network.disconnect_all()
```

Comment:
I haven't seen this problem, but https://docs.fabfile.org/en/2.5/api/connection.html#module-fabric.connection says:

> Connections do not always need to be explicitly closed; much of the time, Paramiko’s garbage collection hooks or Python’s own shutdown sequence will take care of things. However, should you encounter edge cases (for example, sessions hanging on exit) it’s helpful to explicitly close connections when you’re done with them.
>
> This can be accomplished by manually calling close, or by using the object as a contextmanager:

I think this might be worth doing since it seems pretty easy and avoids the problem of hanging connections. What do you think?

Reply:
seems super worth it, agreed.